docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -93,15 +97,16 @@ public static void Example()
                 DataViewSchema schema;
                 using (var file = File.OpenRead("model.zip"))
                     loadedModel = mlContext.Model.Load(file, out schema);
+                // the schema in line 99 and the shuffledFullImagesDataset.Schema in line 93 don't have
+                // the same annotations.
 
                 EvaluateModel(mlContext, testDataset, loadedModel);
 
                 VBuffer<ReadOnlyMemory<char>> keys = default;
                 loadedModel.GetOutputSchema(schema)["Label"].GetKeyValues(ref keys);
```

Comment:
shouldn't need to do all this if you convert predicted label column from Key to Value. It will output string type that will contain class names that corresponds to key indices.